src/pymordemos/phlti.py

```diff
@@ -140,7 +140,9 @@ def main(
     eps = 1e-12
     S += np.eye(S.shape[0]) * eps
 
-    fom = PHLTIModel.from_matrices(J, R, G, S=S, Q=Q, solver_options={'ricc_pos_lrcf': 'slycot'})
+    fom = PHLTIModel.from_matrices(J, R, G, S=S, Q=Q, solver_options={
+        'ricc_pos_lrcf': 'slycot', 'ricc_pos_dense': 'slycot'
+    })
```

Comment:
It might be possible to remove `solver_options`, now that `pymess` is deprecated.

Reply:
No, I'd rather leave the `solver_options`, since `scipy` fails for the particular model.